src/main/java/org/cryptomator/ui/fxapp/FxApplication.java

```diff
@@ -29,9 +29,10 @@ public class FxApplication {
 	private final FxApplicationStyle applicationStyle;
 	private final FxApplicationTerminator applicationTerminator;
 	private final AutoUnlocker autoUnlocker;
+	private final FxFSEventList fxFSEventList; //not unused! By injecting it here, the object gets initiated the service starts
```

Comment:
me is not happy.

ok for now, but we might eventually want to create a meta background task class, which is the only thing injected in the fx main app.

Reply:
I changed it to a manual start. To reduce technical debt. (see https://github.com/cryptomator/cryptomator/pull/3780/commits/bb48551aacc814251276a292718a9da538cfcfcf)